spring-boot-tests/spring-boot-smoke-tests/spring-boot-smoke-test-data-neo4j/src/main/java/smoketest/data/neo4j/CustomerRepository.java

```diff
@@ -16,10 +16,10 @@
 
 package smoketest.data.neo4j;
 
-import java.util.List;
-
 import org.springframework.data.neo4j.repository.Neo4jRepository;
 
+import java.util.List;
```

Comment:
This change does not seem necessary either. I've seen a few reformat in that vein so running the build should be enough to fix them.